tests/testthat/testdata/test_workflow.yaml

```diff
@@ -0,0 +1,54 @@
+meta:
+  MetricID: kri0001
+  File: kri0001.yaml
+  GroupLevel: siteid
+  Abbreviation: AE
+  Metric: Adverse Event Rate
+  Numerator: Adverse Events
+  Denominator: Days on Study
+  Model: Normal Approximation
+  Score: Adjusted Z-Score
+  vThreshold:
```

Comment:
since this draft PR was written, `vThreshold` has been updated to be a string, and the `ParseThreshold` step has been added to all `workflow/metrics` yamls. update `test_workflow.yaml` to match `kri0001.yaml`

Reply:
I directly used `kri0001.yaml` (and tweaked other code to match) so we don't have to duplicate the file.